job_definitions/clean_and_migrate_db_dump.yml

```diff
@@ -0,0 +1,101 @@
+- job:
+    name: clean-and-migrate-db-dump
+    display-name: Clean and migrate database dump
+    project-type: pipeline
+    description: Takes the latest production database dump, cleans it, and migrates to target stage. Also Google Drive.
+    concurrent: false
+    triggers:
+      - timed: "0 4 * * 0"
+    parameters:
+      - extended-choice:
+          name: TARGET_STAGE
```

Comment:
Same note as for the script, calling this `TARGET` feels like a way to avoid some of the assumptions built into `STAGE`